### PR TITLE
first working draft for shields v2

### PR DESCRIPTION
**Pull request by cezaraugusto**
for checking only the interface see https://brave-ui-jffjpwzgeg.now.sh under `Feature Components -> Shields`.

Specs:
**shields up**: https://share.goabstract.com/2663a4f8-953d-4612-a337-806e6946aa9e
**shields down** https://share.goabstract.com/ed8ca2c9-75c6-42c9-afd3-3d90f3711f0a

### CODE REVIEW

**Comment by karenkliu** [comment]
Hi @cezaraugusto please see my design QA feedback attached
shields down
shields up

File: stories/features/shields/components/header.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+export default class ShieldsHeader extends React.PureComponent<Props, {}> {
+  render () {
+    const { enabled } = this.props
+    return (
+      <Header id='braveShieldsHeader' enabled={enabled}>
+        <HeaderToggle enabled={enabled}>
+          <Label size='medium'>
+            {locale.shields}&nbsp;
```
Comment (ryanml) [inline stories/features/shields/components/header.tsx]: Any way we can avoid using `&nbsp` ? :)

File: stories/features/shields/story.tsx [hunk 2]
```diff
@@ -8,8 +8,9 @@ import { storiesOf } from '@storybook/react'
...
 
 // Components
-import BraveShields from './panel/index'
+import Shields from './index'
 
 storiesOf('Feature Components/Shields', module)
   .addDecorator(centered)
-  .add('Page', () => <BraveShields />)
+  .add('Enabled', () => <Shields enabled={true} />)
+  .add('Disabled', () => <Shields enabled={false} />)
```
Comment (ryanml) [inline stories/features/shields/story.tsx]: Can we just use a knob for `enabled` or is there a reason we have two concepts for this one?

File: stories/features/shields/components/header.tsx (hunk 1 above)
Comment (cezaraugusto) [reply stories/features/shields/components/header.tsx]: ya done

File: stories/features/shields/story.tsx (hunk 2 above)
Comment (cezaraugusto) [reply stories/features/shields/story.tsx]: no special reason, added the knob and toggle state

**Comment by ryanml** [comment]
@cezaraugusto looks great! can you squash commits? then i can approve

**Pull request merged**